test/test_metrics.py

```diff
@@ -72,6 +72,7 @@ def test_short_metrics_report_custom_list(self):
     xla_device = xm.xla_device()
     t1 = torch.tensor(100, device=xla_device)
     t2 = t1 * 2
+    t1 += 2
```

Comment:
why?

Reply:
`t1` will be ignored by the `mark_step`(as an output) after my change since it is already a device data. I think one of the metrics checks was checking something related to t1 so I quickly brought it back to the graph as a output.